src/core/prototypes/links/utils.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ChartClass } from "../charts";
+import { LinksProperties } from "./index";
+import { PolarPlotSegment } from "../plot_segments";
+
+export function shouldShowCloseLink(
+  parent: ChartClass,
+  linkProperties: LinksProperties,
+  close?: boolean
```

Comment:
What does this parameter do? It looks as if it is a flag to force the link closed or that the user set the property to close the link. If so, please adjust the naming of this parameter to be more verbose (e.g., `userClose` or `forceClose`.

Reply:
Fixed, thanks!